package.json

```diff
@@ -6,7 +6,7 @@
     "dev": "next dev -p ${CADENCE_WEB_PORT}",
     "build": "NODE_ENV=production && next build",
     "start": "next start -p ${CADENCE_WEB_PORT-8088}",
-    "lint": "next lint",
+    "lint": "next lint --fix",
```

Comment:
Is there a reason behind adding --fix in the lint command ?
This command run with build process so it is better if it can show if the code is building with lint problems.
Because what i expect with this flow is issues will be fixed while building the project in CI and no errors will be shown, while the code will remain with the lint issues in codebase.

What we can do is provide the developer another command to fix lint issues before pushing and maybe also allow vs code to fix lint issues on save

Reply:
Got it. I've removed the flag.
It should be easy enough for a dev to pass the flag themselves anyway so I haven't added a separate command for it.